operations/commit_queue.go

```diff
@@ -20,6 +20,7 @@ const (
 	itemFlagName        = "item"
 	pauseFlagName       = "pause"
 	resumeFlagName      = "resume"
+	commitsFlagName     = "commits"
```

Comment:
Should the new flag be added for regular patches as well?

Reply:
No, I think handling that is out of scope.